### PR TITLE
Bump up online-shop to version v0.3.9

### LINKED ISSUE #227: Bump up Online-shop version to v0.3.8
**Describe the enhancement**
Google released a new version of their online-shop service. Update to the recent version

### LINKED ISSUE #245: Online Shop fail to build
**Describe the bug**
Since the new runners work again we can see that the online shop application fail to build.
https://github.com/ease-lab/vSwarm/actions/runs/2878579648

### PR DESCRIPTION
**Pull request by dhschall**
Closes #227 
Fixes #245

### CODE REVIEW

**Comment by dhschall** [comment]
Fix online-shop functions

File: .github/workflows/e2e-online-shop.yml [hunk 1]
```diff
@@ -206,34 +202,34 @@ jobs:
           go build ./test-client.go
 
           NODEPORT=80
-          url=$(kubectl get kservice ${{ matrix.service }} -o url | awk '$1 {sub(/http\:\/\//,""); print $1}')
+          url=$(kubectl get kservice ${{ matrix.service }} | awk '$2 ~ /http/ {sub(/http\:\/\//,""); print $2}')
+
+          ./test-client --addr $url:$NODEPORT --name "Example text for AES CI"
```
Comment (ustiugov) [inline .github/workflows/e2e-online-shop.yml:207]:
```suggestion
          ./test-client --addr $url:$NODEPORT --name "Example text for CI"
```

File: benchmarks/online-shop/README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ For functionality details refer to the upstream repository ([Online Boutique](ht
 
 
 ## Porting to serverless kernel
-We took the 9 microservices implementing the functionality of the application and integrated them into vSwarm. The current version of the sources is **[v0.3.7](https://github.com/GoogleCloudPlatform/microservices-demo/releases/tag/v0.3.7)**.
+We took the 9 microservices implementing the functionality of the application and integrated them into vSwarm. The current version of the sources is **[v0.3.9](https://github.com/GoogleCloudPlatform/microservices-demo/releases/tag/v0.3.9)**.
 > Note: To upgrade to another version see the `update_remote_files.sh` script.
 
 > Warning: This is just a script for your convince. No guarantee that every will work afterwards.
```
Comment (ustiugov) [inline benchmarks/online-shop/README.md:18]:
```suggestion
> Warning: This is just a script for your convenience. No guarantee that everything will work afterwards.
```

**Review by ustiugov — CHANGES_REQUESTED** [review]
LGTM but please fix the typos.

**Comment by dhschall** [comment]
Spelling is fixed and also your suggestion is applied.
Can we merge?

**Pull request merged**